### ISSUE TITLE
Parent/Task does not always render correct

### ISSUE THREAD

**Issue by MagicMads** [Microsoft/powerbi-visuals-gantt#115]
I am using the Gantt Custom Visual to visualize Surgeries and their sub-steps. The Gantt Chart not showing the data correctly - perhaps because the sub-steps (Tasks) are equally named?

[Gantt Parent:Child Error.docx](https://github.com/Microsoft/powerbi-visuals-gantt/files/1938043/Gantt.Parent.Child.Error.docx)

**Comment by helen508light** [comment]
Hi @MagicMads 
Thanks for you feedback.
Could you also send an .pbix report to pbicvsupport@microsoft.com ?

**Comment by MagicMads** [comment]
Yes - when will you release next? It has been quite a while since last release - /Mads

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by pbicvsupport** [comment]
@MagicMads 
Unfortunately, we can't provide an exact ETA for now.

Waiting for your  .pbix report to pbicvsupport@microsoft.com
